### PR TITLE
Fix build on Xcode 12

### PR DESCRIPTION
**Pull request by rickpasetto**
Uses script (found in https://github.com/Carthage/Carthage/issues/3019#issuecomment-665136323) to work around Xcode 12 build issue with carthage.

### CODE REVIEW

File: Scripts/carthage.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+xcconfig=$(mktemp /tmp/static.xcconfig.XXXXXX)
+trap 'rm -f "$xcconfig"' INT TERM HUP EXIT
+
+# For Xcode 12 make sure EXCLUDED_ARCHS is set to arm architectures otherwise
+# the build will fail on lipo due to duplicate architectures.
+echo 'EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_simulator__NATIVE_ARCH_64_BIT_x86_64__XCODE_1200 = arm64 arm64e armv7 armv7s armv6 armv8' >> $xcconfig
+echo 'EXCLUDED_ARCHS = $(inherited) $(EXCLUDED_ARCHS__EFFECTIVE_PLATFORM_SUFFIX_$(EFFECTIVE_PLATFORM_SUFFIX)__NATIVE_ARCH_64_BIT_$(NATIVE_ARCH_64_BIT)__XCODE_$(XCODE_VERSION_MAJOR))' >> $xcconfig
+
+export XCODE_XCCONFIG_FILE="$xcconfig"
```
Comment (darinkrauss) [inline Scripts/carthage.sh:18]: [nit] I prefer all cap environment variables (it calls them out), plus always surrounding with `{}`. Also, `xcconfig` seems too common and I could envision seeing a naming collision. Perhaps `XCCONFIG_TEMP_OVERRIDE` or something like that?

**Review by darinkrauss — APPROVED** [review]
One nit, but LGTM! 🏝️

File: Scripts/carthage.sh (hunk 1 above)
Comment (rickpasetto) [reply Scripts/carthage.sh:18]: This was literally copy/pasted from https://github.com/Carthage/Carthage/issues/3019#issuecomment-665136323.  :)

Comment (darinkrauss) [reply Scripts/carthage.sh:18]: Ah, then I'd add a comment at the top that includes that link.

**Pull request merged**